### ISSUE TITLE
MOBI/AWZ3 Files Not Counted as Attachments

### ISSUE THREAD

**Issue by eshao** [mickstar/Zoo-For-Zotero#79]
Only EPUBS or PDFs are counted as attachments, thus, when clicking on MOBI or AZW files, they do not open.

**Comment by mickstar** [comment]
I will fix this soon

**Comment by eshao** [comment]
Thanks, really appreciate it! 

Here is a list of file suffixes to support as attachments in case you need to whitelist specific ones:
- MOBI
- AZW
- AZW3
- AZW4
- KFX
- PRC

**Comment by eshao** [comment]
It looks like you've merged in some PRs that address this. It would be great if you could cut a release if it is ready!

**Comment by mickstar** [comment]
Yes, I apologize but it's been a busy few weeks for me and I was passively
testing the build. I will try and push it soon

**Comment by eshao** [comment]
Awesome to hear and appreciate all your great work with the app.
